doc/23_Installation_and_Upgrade/07_Updating_Pimcore/12_V10_to_V11.md

```diff
@@ -1,118 +1,181 @@
-You can call `bin/console doctrine:migrations:migrate` at any given time manually or in your deployment pipeline.
-
-If you are sure you can run all available migrations including bundles and your app-specific migrations after `composer update`, just include the following part in your `composer.json`.
-```json
-"post-update-cmd": [
-    "./bin/console doctrine:migrations:migrate"
-]
+### Migrate System Settings
+
+System Settings are using LocationAwareConfigRepository in Pimcore 11. To keep your specific settings, please follow steps below: 
+
+#### Appearance & Branding 
+Appearance & Branding settings will be separated from the System settings in Pimcore 11 and stored in `var/config/admin_system_settings/admin_system_settings.yaml` by default. Please copy every relevant settings from `var/config/system.yaml` into `var/config/admin_system_settings/admin_system_settings.yaml` to keep your settings. 
+
+To save these settings into the settings store, you can use following configuration:
+```yaml
+pimcore_admin:
+    config_location:
+        admin_system_settings:
+            write_target:
+                type: 'settings-store'
+            read_target:
+                type: 'settings-store'
+```
+
+#### System Settings
+
+Please copy all relevant settings from `var/config/system.yaml` into `var/config/system_settings/system_settings.yaml`.
+
+To save system settings into the settings store, please add following to your configuration:
+```yaml
+pimcore:
+    config_location:
+        system_settings:
+            write_target:
+                type: 'settings-store'
+            read_target:
+                type: 'settings-store'
 ```
 
+### Configuration Storage Settings
+As some features were moved to separate bundles, their configuration storage settings might need to be moved. This is a two step process 
+1) Remove settings prior to update 
+2) Readd settings after extracted bundles are installed
+
+Relevant configs are: 
+```yaml
+# pimcore_custom_reports:
+#  config_location:
+#    custom_reports:
+#      write_target:
+#        type: 'settings-store'
+
+# pimcore_web_to_print:
+#  config_location:
+#    web_to_print:
+#      write_target:
+#        type: 'settings-store'
+
+# pimcore_static_routes:
+#  config_location:
+#    staticroutes:
+#      write_target:
+#        type: 'settings-store'
+```
+
+### Web2Print
+If you are using Web2Print functionality set the flag "Enable Web2Print documents in default documents view" to true to ensure the PimcoreWebToPrintBundle gets installed by default.
+
+If you are not using Web2Print functionality set the flag to false.
+
+## Update Pimcore via Composer
+
+Update Pimcore to Pimcore 11 via composer statement `composer require -W pimcore/pimcore:^11.0 pimcore/admin-ui-classic-bundle`. 
+
+Depending on your setup, it might be necessary to include additional dependencies in the update statement and/or cleanup `composer.lock` file and `vendor` folder.
```

Comment:
```suggestion
Depending on your setup, it might be necessary to include additional dependencies in the update statement and/or to clean up the `composer.lock` file and the `vendor` folder.
```